### PR TITLE
remove unused pkg and update deps

### PR DESCRIPTION
**Pull request by mfshao**
Link to JIRA ticket if there is one: https://ctds-planx.atlassian.net/browse/HP-1589

### Improvements
- improved logic for generating name for array of enums field

### Dependency updates
- update python-json-logger to V2

### CODE REVIEW

File: src/pfb/importers/gen3dict.py [hunk 1]
```diff
@@ -275,7 +275,10 @@ def _array_type(property_type):
         if "description" in property_type:
             enum["name"] = property_type["description"]
         else:
-            enum["name"] = property_type["termDef"][0]["term"]
+            enum["name"] = property_type["term"]["termDef"]["term"]
```
Comment (AlbertSnows) [inline src/pfb/importers/gen3dict.py:278]: I think this block can be rewritten as
```python
enum["name"] = (
    property_type.get("description") or 
    property_type["term"]["termDef"]["term"] or 
    property_type["termDef"][0]["term"])
```

Comment (MichaelLukowski) [inline src/pfb/importers/gen3dict.py:278]: This should exist but should we do something safer like `property_type.get("term", None).get("termDef").get("term")` so we know we get a `None` returned

**Review by AlbertSnows — APPROVED** [review]
Approved with one suggestion. The suggestion is not required to be implemented, but encouraged.

File: src/pfb/importers/gen3dict.py (hunk 1 above)
Comment (mfshao) [reply src/pfb/importers/gen3dict.py:278]: yeah I was suppose to write this using .get() somehow I just forgot 🤦

Comment (AlbertSnows) [reply src/pfb/importers/gen3dict.py:278]: defaulting to None will still be unsafe. If you want to be safe I believe the format would be
`result = property_type.get("term", {}).get("termDef", {}).get("term")`

**Review by AlbertSnows — APPROVED** [review]

**Pull request merged**